src/_Color.sass

```diff
@@ -7,3 +7,4 @@ $color-primary: #9b4dca !default
 $color-secondary: #606c76 !default
 $color-tertiary: #f4f5f6 !default
 $color-quaternary: #d1d1d1 !default
+$color-quinary: #545b61 !default
```

Comment:
I don't think so. Maybe someone will want set other color, not `$color-quaternary, 20%`.